src/Runner.Worker/Handlers/ContainerActionHandler.cs

```diff
@@ -232,5 +229,18 @@ public async Task RunAsync(ActionRunStage stage)
             }
 #endif
         }
+
+        private bool IsDockerFile(string image)
+        {
+            var imageWithoutPath = Data.Image.Split('/').Last();
+            if (imageWithoutPath.StartsWith("Dockerfile") || imageWithoutPath.StartsWith("dockerfile") || imageWithoutPath.EndsWith("Dockerfile") || imageWithoutPath.EndsWith("dockerfile"))
```

Comment:
There is similar code in `ActionManager`, (`.EndsWith("dockerfile")`), do we need this change there too?